Remove references to installing ruby locally (#184)
govuk-one-login/authentication-tech-docs: README.md

* authenticate their users
* verify their users' identity  

## Preview your documentation changes

> To preview any changes and additions you have made to the documentation in a browser, clone this repo and use the [Dockerfile in this repo](Dockerfile) to run a Middleman server on your machine without having to set up Ruby locally.

Reviewer: ```suggestion
To preview your documentation changes in a browser, clone this repository and use the [included Dockerfile](Dockerfile) to run a Middleman server on your machine. This allows you to run the server without setting up Ruby locally. 
```

> This setup has live reload enabled, which means your changes will be applied as you edit files in the source directory. The only exception to this is if you make changes to `config/tech-docs.yml`, you must stop and restart the server to see your changes in the preview. You can stop the server with `Ctrl-C`.

Reviewer: ```suggestion
This Docker container has live reload enabled, which means your changes are applied as you edit files in the source directory. The only exception to this is if you make changes to `config/tech-docs.yml`, you must stop and restart the server to see your changes. You can stop the server with `Ctrl-C`.
```

Run the [helper script](preview-with-docker.sh):

```bash
./preview-with-docker.sh
```

It may take a few minutes to build the docker container, particularly if it is your first time running the script. When the server has finished loading you should then see the following output in the terminal:

```bash
== View your site at "http://localhost:4567", "http://127.0.0.1:4567"
== Inspect your site configuration at "http://localhost:4567/__middleman", "http://127.0.0.1:4567/__middleman"/usr/local/bundle/gems/tilt-2.0.11/lib/tilt/redcarpet.
```

## Making changes to content

> To add or change content, edit the markdown in the `.html.md.erb` files in the `source` folder.

Reviewer: ```suggestion
To add or change content, go to the `source` folder and edit the markdown in the `.html.md.erb` files.
```

> Although a single page of HTML is generated, the markdown is spread across multiple files to make it easier to manage. 

Reviewer: ```suggestion

### Understanding the page structure

Although a single page of HTML is generated, the markdown is spread across multiple files to make it easier to manage. 
```
